packages/icons/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@opentripplanner/icons",
+  "version": "0.0.1",
+  "description": "Icons for otp-ui",
+  "keywords": [
+    "icons"
+  ],
+  "author": "Evan Siroky <evan.siroky@yahoo.com>",
+  "homepage": "https://github.com/opentripplanner/otp-ui#readme",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/opentripplanner/otp-ui.git"
+  },
+  "bugs": {
+    "url": "https://github.com/opentripplanner/otp-ui/issues"
+  },
+  "dependencies": {
```

Comment:
I think core-utils needs to be a peer dependency or else its going to get bundled into each of the individual components. Right as each component ie core-utils is transpiled into es5 I don't think that it efficiently is bundling just the "leg" type with icons for example. I might be wrong about this though, up for discussion.

Reply:
There are currently a lot of helper functions in the core-utils package that this package depends on in order to work. These were extracted from the utils in otp-react-redux, so not a lot of thought or analysis was done to see if these utils are needed just for this package or are needed for other ones as well.

Reply:
Fair enough, but I think it should be a peer dependency shouldn't it?